.circleci/config.yml

```diff
@@ -323,6 +323,8 @@ jobs:
       HELM_VERSION: "v3.1.0"
       KUBEAPPS_DB: "postgresql"
       TEST_UPGRADE: "1"
+      # TODO: Revert this once the latest version is fixed
+      TEST_LATEST_RELEASE: 1
```

Comment:
What's the purpose of this env var? I don't see it's being used in this PR, is it?

Reply:
Yes, this flag is used in the line 163. Basically, if set, we will use the latest tag as image tags for Kubeapps (rather than the image just built). This way we can test the latest release with the current chart (useful for releasing a chart without releasing a new version of kubeapps).

Reply:
Ahhh, got it! I thought it was a new thing in this PR. Thanks for the explanation :)